packages/ui/src/2-resets/_typography.scss

```diff
@@ -10,6 +10,7 @@ html {
   scrollbar-gutter: stable;
   overflow: overlay; // fallback macOS
   overflow-x: hidden;
+  &:where(#{$theme-dark}) {  color-scheme:dark; }
```

Comment:
Is this supposed to show in the kitchen sink? 🧐 This is what it looks like for me

<img width="1435" alt="image" src="https://github.com/appwrite/pink/assets/1477010/b4e7b8d9-e01c-4193-8686-1115c2f9108f">

Reply:
I see it now after the PR was updated!
<img width="1679" alt="image" src="https://github.com/appwrite/pink/assets/1477010/bf6d097d-0f2d-4605-bf6a-626557c67fa9">